test/View/RouteNotFoundStrategyTest.php

```diff
@@ -16,9 +16,12 @@
 use Zend\Mvc\MvcEvent;
 use Zend\Mvc\View\Http\RouteNotFoundStrategy;
 use Zend\View\Model\ViewModel;
+use ZendTest\Mvc\EventManagerIntrospectionTrait;
 
 class RouteNotFoundStrategyTest extends TestCase
 {
+    use EventManagerIntrospectionTrait;
```

Comment:
After reading through the tests, I'm more convinced that the trait should just be a static class